### PR TITLE
Prepare the 0.1.14 release.

### PR DESCRIPTION
**Pull request by jsirois**
Add example docs and fix a download bug seen exercising the example.

### CODE REVIEW

**Comment by jsirois** [comment]
@benjyw and @sureshjoshi if you want to take a look, this is ptex in its 1st final form fully functioning and nearly fully static.

**Comment by sureshjoshi** [comment]
I'll read through the docs this eve to make sure I grok what's going on.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,133 @@
...
+
+In this example Linux x86_64 CPython 3.10 scie, the Linux x86_64 `ptex` binary found in the
+[`ptex` Releases](https://github.com/a-scie/ptex/releases) is directly included. The CPython binary
+distribution from [Python Build Standalone](https://github.com/indygreg/python-build-standalone) is
+not directly included, but instead sourced via `ptex`. This is arranged with a binding command
+configured to run `ptex` passing in the lift manifest itself which `ptex` uses to find the URL of
+the CPython binary distribution when it needs to fetch it.
+
+This results in a `skinny-scie` that is ~5.6MB. On 1st run on the target host you'll see some
+information about the download updating on stderr:
```
Comment (sureshjoshi) [inline README.md:85]: Why `stderr`?

**Comment by sureshjoshi** [comment]
Okay, so the docs all make sense to me. There was one point I think you touched on, which could possibly use an extra sentence or two:

> On subsequent runs on that host, or any host where a scie has already used that CPython
distribution, you'll find:

In this context, if I'm shipping several `ptex`'d utilities that all rely on the same interpreter version (for example) - they would all share a single instance, right - as the fingerprint of the interpreter would be the same? As in, it's kind of a shared resource?

If that's the case, any modification to said shared resource would be propagated to any of the `ptex` executables that use said resource? Or would this be a copy-on-write situation, where running a `ptex` desiring that resource (e.g. interpreter), but only modified ones exist on disk would cause a new one to be downloaded?

File: README.md (hunk 1 above)
Comment (benjyw) [reply README.md:85]: `stderr` makes sense to me, that's where logs normally go.

Comment (jsirois) [reply README.md:85]: Yeah. If you think about pipes in shells it all makes sense. Those pipe stdout by default and the reason you can do interesting things with them is the protocol that programs produce meaningful output on stdout and output for humans on stderr. Stock curl does this with download progress info - stderr; that's why you can `curl foo > foo` and get foo the file with the meaningful data in it while still having your brain entertained by the progress output for humans at the same time.

**Comment by jsirois** [comment]
> In this context, if I'm shipping several ptex'd utilities that all rely on the same interpreter version (for example) - they would all share a single instance, right - as the fingerprint of the interpreter would be the same? As in, it's kind of a shared resource?

Correct - they'd all share the same instance since the interpreters all have the same fingerprint and thus key in the `~/.nce` CAS. They are the same interpreter distribution bit for bit.

> If that's the case, any modification to said shared resource would be propagated to any of the ptex executables that use said resource? Or would this be a copy-on-write situation, where running a ptex desiring that resource (e.g. interpreter), but only modified ones exist on disk would cause a new one to be downloaded?

Here you're slightly off in emphasis. There is no modification at all in this CAS or any other. Its write-once and immutable ever after. If there is a new version of the interpreter distribution it has a new hash and so a new key in the `~/.nce` CAS store. If there is no CAS hit for the new hash key, the CAS entry (interpreter distribution), is extracted from the scie in the case of a traditional scie or downloaded in the case here.

I'll take a spin at some extra language here in the doc.

File: README.md [hunk 2]
```diff
@@ -103,6 +103,12 @@ user    0m0.018s
 sys     0m0.000s
 ```
 
+Since the CPython distribution was already downloaded and extracted (to
+`~/.nce/6c8db44ae0e18e320320bbaaafd2d69cde8bfea171ae2d651b7993d1396260b7` in this case) it is not
```
Comment (jsirois) [inline README.md:107]: I've explained a bit more back in scie-jump here: https://github.com/a-scie/jump/pull/42

File: README.md (hunk 1 above)
Comment (sureshjoshi) [reply README.md:85]: Fair points. Last night while reading, I had it in my head that the downloading process was a bit more fundamental to the output - but yeah, after reading both your comments - it's basically just logs.

**Comment by sureshjoshi** [comment]
> Here you're slightly off in emphasis. There is no modification at all in this CAS or any other. Its write-once and immutable ever after.

👍🏽 

Ah okay, yes, this is what I wasn't quite sure about. I thought it was immutable at first, but then I started remembering the run-once bindings in the `scie` and got confused. 

But on review of the old PR just now - I was mistaken on the mutability of stuff in `~/.nce`

**Review by sureshjoshi — APPROVED** [review]

**Comment by jsirois** [comment]
Thanks for checking this out. On to scie-pants.

**Pull request merged**